opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -175,6 +176,8 @@ def create(
     ) -> "Resource":
         """Creates a new `Resource` from attributes.
 
+        This is supposed to be called by the SDK and not by ResourceDetector instances.
```

Comment:
Nit on "This is supposed to be called by the SDK". I think users would use this directly if they want to pass a resource explicitly like here: https://opentelemetry.io/docs/languages/python/cookbook/#using-multiple-tracer-providers-with-different-resource